xml/admin_operating_services.xml

```diff
@@ -233,4 +233,127 @@ ceph-rbd-mirror@.service
 <screen>&prompt.smaster;salt-run state.orch ceph.restart.ganesha</screen>
   </sect2>
  </sect1>
+ <sect1 xml:id="ceph-cluster-shutdown">
+  <title>Shutdown and Start of the Whole &ceph; Cluster</title>
+
+  <para>
+   There are occasions when you need to stop all &ceph; related services in the
+   cluster in the recommended order, and then be able to simply start them
+   again. For example, in case of a planned power outage.
```

Comment:
I think the two lines might be better reversed.

```
Shutting down and starting the cluster may be necessary such as in the case of a planned power outage.
To stop all &ceph; related services and restart without issue, follow the steps below.
```

Reply:
fixed